src/main/java/mate/academy/spring/model/MovieSession.java

```diff
@@ -18,7 +20,9 @@
     @ManyToOne(fetch = FetchType.LAZY)
     private Movie movie;
     @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "cinema_hall_id")
     private CinemaHall cinemaHall;
+    @Column(name = "show_time")
```

Comment:
`JFYI` Under the hood hibernate will call it the same, even without using of attribute `name`